assets/css/app.css

```diff
@@ -6,4 +6,4 @@
 
 @import 'base/variables.css';
 @import 'base/utils.css';
-@import 'components/how_it_works.css'
\ No newline at end of file
+@import 'components/how_it_works.css';
```

Comment:
new line

Reply:
turiu omeny ne virš paskutinio import be po :smile: 
Visuose failuose paskutinė eilutė turi būti tuščia :smile: 
```
@import url('https://fonts.googleapis.com/css?family=Poppins:400,500,700&display=swap');1

@tailwind base;
@tailwind components;
@tailwind utilities;

@import 'base/variables.css';
@import 'base/utils.css';

@import 'components/how_it_works.css';
(last row, must be empty)
```

Reply:
p.s. jeigu matai tą raudoną simbolį :no_entry_sign: tai reiškias, kad po paskutinės eilutės nėra tuščios eilutės :sweat_smile: